tutorials/kubernetes-demo/manifests/prometheus-helm-chart-with-sidecar.yaml

```diff
@@ -0,0 +1,154 @@
+To deploy thanos sidecar along with Prometheus using official helm chart - just run the next command, putting the values to a file `values.yaml` and changing `--namespace` value beforehand:
```

Comment:
This is definitely not a yaml file. It contains yaml, sure but it's not a YAML not a manifest either.

Can we put it in some new directory like `tutorials/kubernetes-helm/README.md` ? (: I think that would make more sense.

Reply:
@bwplotka changes applied